clients/web/test/spec/modelSpec.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Start the girder backbone app.
+ */
+girderTest.startApp();
+
+describe('Test the model class', function () {
+    var lastRequest, triggerRestError = false, requestCount = 0,
+        windowAlert, lastAlert, alertCount = 0;
+
+    beforeEach(function () {
+        girder.rest.mockRestRequest(function (opts) {
+            requestCount += 1;
+            lastRequest = opts;
+            var resp = $.Deferred();
+            if (triggerRestError) {
+                resp.reject('err');
+            } else {
+                resp.resolve({});
+            }
+            // The jqXHR response of a rest request needs an error function
+            resp.error = resp.fail;
+            return resp;
+        });
+
+        windowAlert = window.alert;
+        window.alert = function (msg) {
```

Comment:
We use alert in a number of places: Collection, AccessControlledModel, GroupView, and Model.  All of them are really for a failure to set up a model properly (thus to inform the developer and not the user).  Converting them to `console.error` seems fine to me.  Would you like that on this or a different PR?